### ISSUE TITLE
Upholstery Cleaning Services in Braga

### ISSUE THREAD

**Issue by Stephanymtr** [dwyl/home#223]
We want to get two sofas (from piano room and dining room) and one large rug (from playroom) professionally cleaned. I have gathered a couple of companies that provide upholstery cleaning services in Braga. I have also contacted them all and will update this issue with their quotes as they get back to me.

### IMPEC CLEAN 
https://www.impecclean.pt/
☎️ 932270602
Price €215 
They couldn't talk over the phone. They replied to me right away with a voice message on Whatsapp explaining in details their cleaning process... very diligent. They have availability for next week.

### Higiservice - Higienização Profissional de Estofos
https://higiservice.pt/
☎️ 965066084
Price €167
They couldn't talk much on the phone, but they were able to give me a quote. They have availability for 22/02 in the morning.

### LEC
https://www.leader.pt/
☎️ 912198512
Price €
_still waiting_

### CC Clean
https://ccclean.pt/
☎️ 933486957
Price €
_still waiting_

@iteles Let me know if there's any other information you would like to know while we are waiting for the rest of the quotes. Thank you.

**Comment by iteles** [comment]
The only one I've had recommended by someone (I tangentially know) has been https://limpinho.pt/

Could we also get a quote from them so that we can see what the 'recommended' ballpark is?

**Comment by nelsonic** [comment]
@iteles and @Stephanymtr just a quick FYI: we recently got a **`Kaercher WD 6 P`** Wet and Dry vacuum cleaner
(on Cyber Monday special it was **`35% off`**...!)

https://www.kaercher.com/ie/home-garden/wet-and-dry-vacuum-cleaners/wd-6-p-premium-16283780.html
<img width="1409" alt="image" src="https://user-images.githubusercontent.com/194400/219375154-8d710334-9869-4c14-8e8e-c1f4839402d1.png">

got it for cleaning the workshop ... but since it's "Wet and Dry" it can _easily_ be used to deep-clean the sofas.
see: https://www.kaercher.com/my/inside-kaercher/company/newsroom/household-cleaning-tips/how-to-clean-fabric-sofa.html 

Yes, there are more expensive machines that have a steam/spray function. 
Definitely still get quotes from professional cleaning companies. 

But IMO the combination of a good Carpet Cleaning liquid e.g: `KÄRCHER RM519`
https://www.worten.pt/casa/limpeza/produtos-e-acessorios/limpa-carpetes-karcher-rm519-1l-MRKEAN-4039784712669?gclid=EAIaIQobChMI7Y7vqZGa_QIVDg0GAB1IXAtzEAQYASABEgLnevD_BwE
<img width="1236" alt="image" src="https://user-images.githubusercontent.com/194400/219376742-b1dd6b25-651b-4323-984f-61eb704da713.png">

And a bit of effort with the `WD 6 P` would more than do the trick for our sofas.
They are dirty, but thankfully not to the point of needing "professional" intervention. 💭 
I'd much rather pay **`M`** for 6 hours and do it "in-house" ... wouldn't you? 🤷‍♂️ 
That way we could develop an internal "system" for doing this periodically e.g. once every 3 months. ⏳

**Comment by iteles** [comment]
Agreed, I didn't remember we had this! I've never actually seen it 😄

@Stephanymtr  let's agree a day next week when you're coming in for M to come in with you and get this done if she's available and up for it. 

@nelsonic Do you need to go through how this works with Stephany before we leave?

**Comment by nelsonic** [comment]
@iteles you've walked past it several times in the Workshop; so you've seen it. 😉 
Perhaps you just didn't register what it _is_ in your passing. 💭 

As for going through how it works ... I'd recommend that `M` read the instructions: 
https://s1.kaercher-media.com/documents/manuals/raw/000/BTA-5674184-000-03.pdf
We've opened the box to use it in the Workshop. 
But using it "wet" requires reading the instructions. (or watching a video online...) 
e.g: https://youtu.be/brvsMdhSdh4

**Comment by iteles** [comment]
@Stephanymtr How comfortable do you think `M` would be about this?

**Comment by Stephanymtr** [comment]
Limpinho - Sociedade de Limpezas, Lda.
https://limpinho.pt/
☎️ 913382303
Price €246

------
The video looks a lot more helpful than the manual. I think she will be ok in terms of figuring out **how the machine works**.

_Side note: I think the large rug backing is deteriorating. The glue is slowly dissolving making it look like sand is coming through the carpet. I **_think_** the more we clean the rug, the faster the backing will deteriorate. It might be worth looking into fixing this first._

**Comment by iteles** [comment]
I've ordered the liquid and it should arrive tomorrow (Thursday 23rd Feb).

**Comment by Stephanymtr** [comment]
@iteles Is my mom coming over tomorrow or did you want me to ask her availability ? Do you think the liquid will arrive early enough for her to be able to do all the cleaning?

**Comment by nelsonic** [comment]
If we're going to do the upholstery cleaning "in-house" then we need to source the appropriate cleaning liquid ASAP. ⏳

**Comment by iteles** [comment]
@Stephanymtr I texted her this morning to ask if she was available on Friday. I don't think it makes any sense for her to come in tomorrow when it might only arrive at the end of the day 👌🏻

**Comment by Stephanymtr** [comment]
@iteles Cool, I'm glad I asked because she deleted her whatsapp - by mistake. I'll ask her to contact you via SMS for now until we manage to get the app back on her phone.

**Comment by iteles** [comment]
This worked well! Closing for now.
